include/internal/libspdm_common_lib.h

```diff
@@ -78,6 +78,17 @@ typedef struct {
 #endif
 } libspdm_peer_used_cert_chain_t;
 
+typedef struct {
+    uint16_t capabilities;
+    uint16_t key_usage_capabilities;
+    uint16_t current_key_usage;
+    uint32_t asym_algo_capabilities;
+    uint32_t current_asym_algo;
+    uint16_t public_key_info_len;
+    uint8_t assoc_cert_slot_mask;
+    uint8_t public_key_info[LIBSPDM_MAX_PUBLIC_KEY_INFO_LEN];
+} libspdm_key_pair_info;
```

Comment:
it should be `libspdm_key_pair_info_t`

Reply:
Thanks. I have made the format consistent.